src/app/badges-and-levels/page.tsx

```
@@ -0,0 +1,436 @@
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card';
+import { Badge } from '@/components/ui/badge';
+import { Progress } from '@/components/ui/progress';
+import { Skeleton } from '@/components/ui/skeleton';
+
+interface Badge {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Badge'. Consider to delete it or rename it.

'Badge' is defined here:


(lint/suspicious/noRedeclare)